packages/toolkit/CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+## 3.33.0 2024-11-06
+- **Bug Fix** SAM: Save SAM deploy parameters to correct samconfig.toml in project folder
+
```

Comment:
Do not manually add the changelog. Use `npm run newChange`

Reply:
I reverted manual update and replaced with change json object for next release.